lightning/src/routing/scorer.rs

```diff
@@ -86,11 +97,36 @@ pub struct ScoringParameters {
 	pub failure_penalty_msat: u64,
 
 	/// The time needed before any accumulated channel failure penalties are cut in half.
-	#[cfg(not(feature = "no-std"))]
 	pub failure_penalty_half_life: Duration,
```

Comment:
IIUC, could we document that this field isn't used if `no-std` is enabled + that penalties will never decay under `no-std`?